doc/maintaining/cli.rst

```diff
@@ -482,6 +482,19 @@ The result is saved in profile.data.search. To view the profile in runsnakerun::
 You may need to install the cProfile python module.
 
 
+run: Start Development server
+==================================
+
+Usage
+
+.. parsed-literal::
+
+ ckan run --host (-h)          - Set Host
+ ckan run --port (-p)          - Set Port
+ ckan run --reloader (-r)      - Use reloader
```

Comment:
This syntax changed in https://github.com/ckan/ckan/pull/5639 (we have `--disable-reloader` now). Can you update @tino097 ? cheers

Reply:
Done!